config/environments/development.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Serve the client/dist directory
+  config.public_file_server.enabled = true
+  paths['public'] = File.join 'client', 'dist'
```

Comment:
This line is missing from `config/environments/test.rb`. Since we need both of these lines in all environments, we should have them in `config/application.rb` only and remove them from each environment config file.